### PR TITLE
Updated alexa plugin setup documentation to reflect current Alexa dev portal

### PR DESCRIPTION
**Pull request by veryfancy**
Fix for issue https://github.com/nightscout/cgm-remote-monitor/issues/3512

Updated alexa-plugin.md to include n00b-friendly, step-by-step instructions to set up alexa plugin using the current Amazon dev portal.

Left section "Adding Alexa support to a plugin" alone, aside from fixing the headers in it.

### CODE REVIEW

**Comment by PieterGit** [comment]
@aaronjknox or @tanja3981 or @jettechfsr or @winni67 can you review @veryfancy PR.
If it's ok I'll merge it to dev and it will be available with the next Nightscout release.

File: lib/plugins/alexa-plugin.md [hunk 1]
```diff
@@ -1,100 +1,274 @@
...
+- Sign up for a free [Amazon Developer account](https://developer.amazon.com/) if you don't already have one.
+- [Register](https://developer.amazon.com/docs/devconsole/test-your-skill.html#h2_register) your Alexa-enabled device with your Developer account.
+- Sign in and go to the [Alexa developer portal](https://developer.amazon.com/alexa). 
+
+### Create a new Alexa skill
+
+1. Select "Alexa Skills Kit" in the main menu bar.
+1. Click the "Start a Skill" button. This will take you to the Skills console.
+1. Click the "Create Skill" button on the Skills console page.
+1. Name your new skill "Nightscout" (or something else, if you like). Use English (US) as your language. Click "Next".
```
Comment (tanja3981) [inline lib/plugins/alexa-plugin.md]: I'm not really happy about tellling to use language english without further note that the own language can be used if the corresponding translations are available...

**Comment by tanja3981** [comment]
One minor comment, which I would to update in another pull request where I would add some more information about how other languages can be handled and maybe add some note about translations.

However, I think the PR should be merged!

**Comment by PieterGit** [comment]
@veryfancy thanks for the doc update
@tanja3981 thanks for the review. please open a separate PR for the languages improvement

**Pull request merged**